UIKITDZ/UIKITDZ/ViewController.swift

```diff
@@ -8,10 +8,50 @@
 import UIKit
 
 /// ViewController
-class ViewController: UIViewController {
+final class ViewController: UIViewController {
+    
+     let text = GameHelloModel()
```

Comment:
```suggestion
     let text = GameHelloModel()
```
нужно такое же название как и модель